agent/engine/docker_task_engine_test.go

```diff
@@ -579,7 +579,7 @@ func TestTaskTransitionWhenStopContainerTimesout(t *testing.T) {
 			// Since task is not in steady state, progressContainers causes
 			// another invocation of StopContainer. Return a timeout error
 			// for that as well
-			client.EXPECT().StopContainer(gomock.Any()).Return(containerStopTimeoutError),
+			client.EXPECT().StopContainer("containerId").Return(containerStopTimeoutError).AnyTimes(),
```

Comment:
same as above: would it be better to add a TODO here to move this to MinTimes(1) ?